packages/@okta/vuepress-site/code/javascript/okta_auth_sdk/index.md

```diff
@@ -9,11 +9,13 @@ The Okta Auth SDK builds on top of our [Authentication API](/docs/reference/api/
 
 The Okta Auth SDK is used by Okta's [Sign-in Widget](/code/javascript/okta_sign-in_widget/) which powers the default Okta sign-in page. If you are building a JavaScript front end or Single Page App (SPA), the Auth SDK gives you added control and customization beyond what is possible with the Widget.
 
-In this guide you will learn how to use the Auth SDK on a simple static page to:
+In this guide you will learn how to use the Auth SDK to:
 
 - Retrieve and store an OpenID Connect (OIDC) token
 - Get an Okta session
 
+**NOTE**: `@okta/okta-auth-js` version 3.2.2 or above is required to run samples in this guide.
```

Comment:
Docs have used different conventions to add notes. But most of them use "block quotes" AFAIK 
> *Note:* This is a note.

@jakubvul , @susanharper-okta - Any convention of how to write notes?

Reply:
> **Note:** is what we do for a note.

Reply:
>**Note:** This is a note